src/app/docs/toolbar/toolbar-guidelines.component.ts

```diff
@@ -0,0 +1,208 @@
+  {
+    label: 'Text',
+    value: 'Edit'
+  },
+  {
+    label: 'Icon',
+    value: 'fa-pencil'
+  },
+  {
+    label: 'Dropdown',
+    value: 'Optional. Allows users to edit multiple types of records in a content container.'
+  }
+];
+public saveButtonInfo: {label: string, value: string}[] = [
+  {
+    label: 'Text',
+    value: 'Save'
+  },
+  {
+    label: 'Icon',
+    value: 'fa-save'
+  },
+  {
+    label: 'Dropdown',
+    value: 'Optional. Allows users to select multiple choices for how to save a list, such as "Save" and "Save as."'
+  }
+];
+public filterButtonInfo: {label: string, value: string}[] = [
+  {
+    label: 'Text',
+    value: 'Filter'
+  },
+  {
+    label: 'Icon',
+    value: 'fa-filter'
+  },
+  {
+    label: 'Dropdown',
+    value: 'None'
+  }
+];
+public sortButtonInfo: {label: string, value: string}[] = [
+  {
+    label: 'Text',
+    value: 'Sort'
+  },
+  {
+    label: 'Icon',
+    value: 'fa-sort'
+  },
+  {
+    label: 'Dropdown',
+    value: 'Yes'
+  }
+];
+public columnsButtonInfo: {label: string, value: string}[] = [
+  {
+    label: 'Text',
+    value: 'Columns'
+  },
+  {
+    label: 'Icon',
+    value: 'fa-columns'
+  },
+  {
+    label: 'Dropdown',
+    value: 'None'
+  },
+  {
+    label: 'Notes',
+    value: 'Only visible when viewing a list as a grid.'
+  }
+];
+public shareButtonInfo: {label: string, value: string}[] = [
+  {
+    label: 'Text',
+    value: 'Share'
+  },
+  {
+    label: 'Icon',
+    value: 'fa-share'
+  },
+  {
+    label: 'Dropdown',
+    value: 'None'
+  }
+];
+public exportButtonInfo: {label: string, value: string}[] = [
+  {
+    label: 'Text',
+    value: 'Export'
+  },
+  {
+    label: 'Icon',
+    value: 'fa-file-o, fa-file-excel-o, or fa-file-pdf-o'
+  },
+  {
+    label: 'Dropdown',
+    value: 'Optional. Allows users to select from multiple export options.'
+  },
+  {
+    label: 'Notes',
+    value: 'For multiple export options, use the generic fa-file-o icon and a dropdown. For a single export option, use the icon for the file type that the option creates.'
+  }
+];
+public moreButtonInfo: {label: string, value: string}[] = [
+  {
+    label: 'Text',
+    value: 'More'
+  },
+  {
+    label: 'Icon',
+    value: 'fa-ellipsis-h'
+  },
+  {
+    label: 'Dropdown',
+    value: 'Yes'
+  },
+  {
+    label: 'Notes',
+    value: 'Only use the more actions button for two or more items. For a single action, display the action button in the toolbar.'
+  }
+];
+public findBoxInfo: {label: string, value: string}[] = [
+  {
+    label: 'Text',
+    value: 'Find in this list'
+  },
+  {
+    label: 'Icon',
+    value: 'fa-search'
+  },
+  {
+    label: 'Notes',
+    value: 'This search component should have the same interactions and highlights as other contexts where the search component is used.'
+  }
+];
+public simpleFiltersInfo: {label: string, value: string}[] = [
+  {
+    label: 'Used in',
+    value: 'Tiles, record pages, or lists'
```

Comment:
I'd reverse the order here to use alphabetic order.

Reply:
Following up again. Do you think these changes are worth making, or should we just resolve the conversation?